File: backend/platform/lib/models/resume.py
from pydantic import BaseModel


class RegisterResponse(BaseModel):
  ids: list[str]


class ResumeDetails(BaseModel):
  id: str
  base_requirement_satisfaction_score: int
  exceptional_considerations: str
  fitness_score: int


class ListClassifiedResponse(BaseModel):
  very_fit: list[ResumeDetails]
  fit: list[ResumeDetails]
  not_fit: list[ResumeDetails]

File: frontend/src/lib/models/resume.ts
export interface ResumeDetails {
  id: string;
  base_requirement_satisfaction_score: number;
  exceptional_considerations: string;
  fitness_score: number;
}

export interface ListClassifiedResponse {
  very_fit: ResumeDetails[];
  fit: ResumeDetails[];
  not_fit: ResumeDetails[];
}
